lib/hotel/hotel.dart

```diff
@@ -1,14 +1,297 @@
+          pageSnapping: true,
+          itemBuilder: (context, index) {
+            return Stack(
+              children: <Widget>[
+                Container(
+                  height: MediaQuery.of(context).size.height * 0.70,
+                  decoration: BoxDecoration(
+                    image: DecorationImage(
+                      image: ExactAssetImage(
+                        hotelImages[index],
+                      ),
+                      fit: BoxFit.cover,
+                    ),
+                  ),
+                ),
+              ],
+            );
+          },
+          itemCount: hotelImages.length,
+          onPageChanged: (index) {
+            setState(() {
+              _pageViewNotifier.value = index;
+            });
+          },
+        ),
+        Padding(
+          padding: EdgeInsets.only(
+              bottom: MediaQuery.of(context).size.height * 0.45),
+          child: Transform.translate(
+            offset: Offset(0, 175),
+            child: Align(
+              alignment: Alignment.center,
+              child: _displayPageIndicators(hotelImages.length),
+            ),
+          ),
+        ),
+        Align(
+          alignment: Alignment.bottomCenter,
+          child: Container(
+            height: MediaQuery.of(context).size.height * 0.45,
+            width: double.infinity,
+            color: Colors.white,
+          ),
+        ),
+        Padding(
+          padding: const EdgeInsets.all(25.0),
+          child: Scaffold(
+            backgroundColor: Colors.transparent,
+            appBar: AppBar(
+              backgroundColor: Colors.transparent,
+              elevation: 0,
+              leading: IconButton(
+                onPressed: () => Navigator.pop(context),
+                icon: Icon(Icons.close),
+                color: Colors.white,
+              ),
+              actions: <Widget>[
+                IconButton(
+                  onPressed: () {},
+                  icon: Icon(Icons.today),
+                  color: Colors.white,
+                ),
+                IconButton(
+                  onPressed: () {},
+                  icon: Icon(Icons.share),
+                  color: Colors.white,
+                ),
+                IconButton(
+                  onPressed: () {},
+                  icon: Icon(Icons.more_vert),
+                  color: Colors.white,
+                ),
+              ],
+            ),
+            floatingActionButton: Padding(
+              padding: EdgeInsets.only(
+                  bottom: MediaQuery.of(context).size.height * 0.36),
+              child: FloatingActionButton(
+                  onPressed: () {},
+                  backgroundColor: Colors.white,
+                  child: Icon(
+                    Icons.bookmark_border,
+                    color: Colors.pink,
+                  )),
+            ),
+            floatingActionButtonLocation: FloatingActionButtonLocation.endFloat,
```

Comment:
**Unfortunately** wrong use of _FloatingActionButton_ widget
  - it must be floating all time not static
  - use IconButton widget instead